### DISCUSSION TITLE
.RedirectTo not being respected at Email template on href

### DISCUSSION THREAD

**Discussion by WilliamCunhaCardoso** [supabase/supabase#33013] [Questions]
# Bug report

- [ ] I confirm this is a bug with Supabase, not with my own application. 
- [ ] I confirm I have searched the [Docs](https://docs.supabase.com), GitHub [Discussions](https://github.com/supabase/supabase/discussions), and [Discord](https://discord.supabase.com). 

## Describe the bug

Consider you have the following setup:

**URL Configuration**
Site URL: `potato.com`
Additional URL: `dev://app/reset`

**Email Template:**
```
--First Part
<p><a href="{{ .RedirectTo }}?token={{.Token}}">Reset Password</a></p>

<p><a href="{{ .RedirectTo }}">Only Redirect</a></p>

-- Second Part
<p>Token: "{{ .Token }}"</p>

<p>Redirect To: "{{ .RedirectTo }}"</p>

<p>URL Mounted: "{{ .RedirectTo }}?token={{.Token}}"</p>
```

**Description**

What happens is, **.RedirectTo** is respected at second part. So, at my Flutter application, I call:
```
 supabaseClient.auth.resetPasswordForEmail(
      email,
      redirectTo: "dev://app/resetPassword",
    );
```

It works very fine at paragraphs. But, on the first part, where it should matter, it is not respected. It somehow points out to opened page, like the email provider you are seeing the email.

What I am missing here? Why at link if is not respected, but at paragraphs it is?

## To Reproduce

Setup Additional URL for mobile, like:
Additional URL: `dev://app/reset`

At flutter project, call:
```
supabaseClient.auth.resetPasswordForEmail(
      email,
      redirectTo: "dev://app/reset",
    );
```

At Email Template, to reset password template:
```
--First Part
<p><a href="{{ .RedirectTo }}?token={{.Token}}">Reset Password</a></p>

<p><a href="{{ .RedirectTo }}">Only Redirect</a></p>

-- Second Part
<p>Token: "{{ .Token }}"</p>

<p>Redirect To: "{{ .RedirectTo }}"</p>

<p>URL Mounted: "{{ .RedirectTo }}?token={{.Token}}"</p>
```

## Expected behavior

The links being equal to what is on the paragraph:

dev://app/reset?token=1234

## System information

- OS: [e.g. macOS, Windows]
- Browser (if applies) [e.g. chrome, safari]
- Version of supabase-js: [e.g. 6.0.2]
- Version of Node.js: [e.g. 10.10.0]

**Comment by Hallidayo** [comment]
Hi everyone, due to inactivity on this issue I've moved the issue over to discussions/enhancements.
